OpenSim/Examples/Moco/example2DWalking/CMakeLists.txt

```diff
@@ -13,3 +13,9 @@ file(COPY ${EXAMPLE_2D_WALKING_FILES}
         "${CMAKE_BINARY_DIR}/Bindings/Java/Matlab/examples/Moco/example2DWalking")
 install(FILES ${EXAMPLE_2D_WALKING_FILES}
     DESTINATION "${OPENSIM_INSTALL_MATLABEXDIR}/Moco/example2DWalking")
+
+file(COPY ${EXAMPLE_2D_WALKING_FILES}
```

Comment:
I assume that you tested that this CMake code correctly installs the reference files in the Python example directory?

Reply:
I can confirm that it installs the reference files except the Geometry folder. The CMake file doesn't seem to mention anything about the Geometry folder for the equivalent matlab examples, so I am wondering if there is another file that needs to be tweaked?

Reply:
There is no Geometry folder for this example, so no worries.